modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/block/BlockStorage.scala

```diff
@@ -44,7 +46,64 @@ class BlockStorage[F[_]: Sync: Random](blocks: MapRef[F, ProofsHash, Option[Stor
       case Some(_: WaitingBlock) => (AcceptedBlock(hashedBlock).some, ().asRight)
       case other                 => (other, BlockAcceptanceError(hashedBlock.proofsHash, other).asLeft)
     }.flatMap(_.liftTo[F])
-      .flatMap(_ => updateParentUsages(hashedBlock))
+      .flatMap(_ => addParentUsages(hashedBlock))
+
+  def adjustToMajority(
+    toAdd: Set[(Hashed[DAGBlock], NonNegLong)] = Set.empty,
+    toMarkMajority: Set[(ProofsHash, NonNegLong)] = Set.empty,
+    acceptedToRemove: Set[ProofsHash] = Set.empty,
+    obsoleteToRemove: Set[ProofsHash] = Set.empty,
+    toReset: Set[ProofsHash] = Set.empty
+  ): F[Unit] = {
+    val addMajorityBlocks = toAdd.toList.traverse {
```

Comment:
a rule of thumb is to write it as `def` instead of `val`. it doesn't matter because from FP standpoint both are programs that are pure but I read some time ago that it's better to use `def` wherever possible to emphasise that

Reply:
Ok, let's do it like that.